torchao/utils.py

```diff
@@ -526,7 +585,24 @@ def fill_defaults(args, n, defaults_tail):
 def _torch_version_at_least(min_version):
     return is_fbcode() or version("torch") >= min_version
 
+
 TORCH_VERSION_AFTER_2_5 = _torch_version_at_least("2.5.0.dev")
 TORCH_VERSION_AFTER_2_4 = _torch_version_at_least("2.4.0.dev")
 TORCH_VERSION_AFTER_2_3 = _torch_version_at_least("2.3.0.dev")
 TORCH_VERSION_AFTER_2_2 = _torch_version_at_least("2.2.0.dev")
+
+
+def add_to_globals(cls: Any):
+    """Add a class to the global namespace for serialization
+    This is needed to make sure that a tensor subclass can be serialized
+
+    Example:
+    ```
+        class MyTensor(TorchAOBaseTensor):
+            pass
+
+        add_to_globals(MyTensor)
+    ```
+    """
+    if TORCH_VERSION_AT_LEAST_2_5:
+        torch.serialization.add_safe_globals([cls])
```

Comment:
or just inline?

Reply:
I think we are going to have to add this in many places I thought it would be nice to have a helper and point people to making new subclasses to this utility

Reply:
feels a bit overkill to save one line, but if you want to make a util it would be better to align with the signature of https://pytorch.org/docs/stable/notes/serialization.html#torch.serialization.add_safe_globals I think